inlong-agent/agent-core/pom.xml

```diff
@@ -84,5 +84,16 @@
             <groupId>io.prometheus</groupId>
             <artifactId>simpleclient_httpserver</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.typesafe.akka</groupId>
+            <artifactId>akka-stream_2.11</artifactId>
+            <scope>compile</scope>
```

Comment:
The default scope is `compile`, so this line can be removed.

Reply:
done

Reply:
done